batch_inference_v2/batch_inference_v2.ipynb

```diff
@@ -62,8 +62,7 @@
     "    \n",
     "* **model_path**: `Union[str, mlrun.DataItem]`\n",
     "    \n",
-    "    The model Store path. Can be provided as an input or as a parameter.\n",
-    "    If `endpoint_id` of existing model endpoint is provided, make sure that it has a similar model store path, otherwise the drift analysis won't be triggered.\n",
+    "    Model store uri (should start with store://). Can be provided as an input or as a parameter. To generate a valid model store URI, please log the model before     running this function. If `endpoint_id` of existing model endpoint is provided, make sure that it has a similar model store path, otherwise the drift analysis     won't be triggered.\n",
```

Comment:
Can be provided as an input or as a parameter => Provided as an input (DataItem)